src/resources/games/controller.js

```diff
@@ -1,3 +1,67 @@
 const prisma = require("../../utils/db");
 
-module.exports = {};
+async function getUserGames(req, res) {
+  console.log({ USER: req.user });
+  const foundUser = req.user;
+
+  try {
+    const games = await prisma.game.findMany({
+      where: {
+        users: {
+          some: {
+            user: {
+              id: parseInt(foundUser.id),
+            },
+          },
+        },
+      },
+      include: {
+        users: {
+          include: {
+            user: {
+              select: {
+                username: true,
+              },
+            },
+          },
+        },
+      },
+    });
```

Comment:
This looks overly complicated!

Reply:
If you are attempting to get the user's games you want to find the user and `include` games. Prisma will do the work for you because you've declared the relationships in the schema.